docs/MigrationGuide.md

```diff
@@ -0,0 +1,118 @@
+# Migration Guide from unleash-android-proxy-sdk
+
+This guide provides detailed steps for migrating your project from the Unleash Android Proxy SDK version to the newer Unleash Android SDK.
+
+We will focus on the [previous sample application](https://github.com/Unleash/unleash-android-proxy-sdk/tree/main/samples/android) and specifically highlighting the changes from this pull request: https://github.com/Unleash/unleash-android-proxy-sdk/pull/83
+
+## Benefits of Migrating
+
+This version of the Unleash Android SDK introduces several improvements, including:
+- No need for SLF4J dependency and the usage of the native Android logging system.
```

Comment:
It's unclear whether it's "no need for [...] the usage of the native Android logging system", or not. I suspect that the android logging system replaces SLF4J, though. How about this?

```suggestion
- No need for the SLF4J dependency. Instead, use the native Android logging system.
```

or

```suggestion
- Uses the native Android logging system instead of SLF4J.
```